UsageDoc/CA_UsageDocument.md

```diff
@@ -241,9 +241,9 @@ Description for the settings in `appSettings.json` file
 
 |S.No| Argument name   |Description  | Is it Mandatory    | Example |
 |--|--|--|--|--|
-| 1 |--packagefilepath   | Path to the package-lock.json file or to the directory where the project is present in case we have multiple package-lock.json files.                                      |Yes  | D:\Clearing Automation|
+| 1 |--packagefilepath   | Path to the package-lock.json file or to the directory where the project is present in case we have multiple package-lock.json files.                                      |Yes  | D:\Clearing Automation, For Docker run /mnt/Input|
 | 2 |--cycloneDxbomfilePath | Path to the cycloneDx BOM file. This should not be used along with the package file path(arg no 1).Please note to give only  one type of input at a time.                           |No if the first argument is provided| D:\ExternalToolOutput|
-| 3 |--bomfolderpath | Path to keep the generated boms  |  Yes     | D:\Clearing Automation\BOM
+| 3 |--bomfolderpath | Path to keep the generated boms  |  Yes     | D:\Clearing Automation\BOM, For Docker run /mnt/Output
```

Comment:
we are giving the mandatory value in the example column . is it okay ?

Reply:
This can be considered as a reference by the user. But in the docker file we have specifies these values for mounting the input